src/app/state/rewardSlice.ts

```diff
@@ -83,53 +90,55 @@ type NonFungibleResource = NonFungibleResourcesCollectionItem & {
 // when fetching receipts for rewards, you need to fetch receipts for all pairs (not just the DEXTER/XRD pair).
 // You can use the existing alphadex api endpoint: /account/orders to get the orders for each pair.
 export const fetchReciepts = createAsyncThunk<
-  undefined, // Return type of the payload creator
+  string[], // Return type of the payload creator
   undefined, // argument type
   {
     state: RootState;
   }
->("rewards/fetchReciepts", async (_, thunkAPI) => {
-  const dispatch = thunkAPI.dispatch;
+>("rewards/fetchReciepts", async () => {
   const rdt = getRdt();
-  if (!rdt) return;
+  if (!rdt) {
+    throw new Error("RDT initialization failed");
+  }
+
+  // Question from @dcts: should those be really taken from .env
+  // or are we fetchnig those too with fetchAddresses ?
   const claimComponentAddress = process.env.NEXT_PUBLIC_CLAIM_COMPONENT;
```

Comment:
Are those addresses supposed to be inside the .env or should those also be fetched by fetchAddresses?